frontend/src/components/pages/recommandationForm/RecommendationForm.js

```diff
@@ -12,13 +12,16 @@ function RecommendationForm({handleSubmit,handleDataInput,wineConfiguration}) {
    const nextStep = (value) => {
       setStep(parseInt(value))
    }
+   const stepBack=(value)=>{
+       setStep(parseInt(value))
+   }
```

Comment:
Somewhere, either here or in the Step.js you would need to check that nextStep isn't false if I understand your logic correctly.